website/intro/getting-started/build.html.md

```diff
@@ -25,9 +25,6 @@ For the getting started guide, we'll only be using resources
 which qualify under the AWS
 [free-tier](https://aws.amazon.com/free/),
 meaning it will be free.
-If you already have an AWS account, you may be charged some
-amount of money, but it shouldn't be more than a few dollars
-at most.
```

Comment:
Is there a reason for removing this part?

Reply:
Because the warning below repeats the same.